packages/rum-core/test/common/apm-server.spec.js

```diff
@@ -423,6 +423,25 @@ describe('ApmServer', function() {
     expect(result).toEqual([expected])
   })
 
+  it('should pass correct payload to filters', () => {
+    configService.setConfig({
+      serviceName: 'serviceName'
+    })
+    let type = ''
+    configService.addFilter(function(payload) {
+      expect(payload[type]).toBeDefined()
+      return payload
+    })
+
+    type = 'errors'
+    let result = apmServer.sendErrors([{ test: 'test' }])
```

Comment:
you would have to use await since sendErrors returns a promise. Might be hard to catch these bugs since errors would be logged later on.